ruby-challenges.rb

```diff
@@ -3,30 +3,102 @@
 
 # --------------------1) Create a method that takes in an array of words and a single letter and returns an array of all the words containing that particular letter. Use the test variables provided.
 
+# Pseudocode 1.) Create a method called containsLetter
+# 2.) containsLetter will take in an array and a string as an argument.
+# 3.) create an empty array called answer
+# 4.) use .each to iterate over the array array argument
+# 5.) use a condition statment and pass in value through the pipes.
+# 6.) use .include? method to check array argument includes the string argument
+# 7.) if the condition is met shovel the value at index into answer
+
 beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
 
 letter_o = 'o'
 # Expected output: ['coffee', 'soda water']
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def containsLetter (array, string)
+    answer = []
+    array.each do |value|
+        if value.include?(string)
+        answer << value
+        end
+     end
+    return answer
+end
```

Comment:
Well done! How can you refactor this to not need the answer variable? (Check out `.select` as one example among many of how you could do this)